app/mbedtls/app/espconn_mbedtls.c

```diff
@@ -423,30 +423,24 @@ static bool mbedtls_handshake_result(const pmbedtls_msg Threadmsg)
 static void mbedtls_fail_info(espconn_msg *pinfo, int ret)
 {
 	pmbedtls_msg TLSmsg = NULL;
-	lwIP_REQUIRE_ACTION(pinfo,exit,);
+	lwIP_REQUIRE_ACTION(pinfo,exit,ret = ERR_ARG);
 	TLSmsg = pinfo->pssl;
-	lwIP_REQUIRE_ACTION(TLSmsg,exit,);
-
```

Comment:
Please do not apply the next hunks, from here to line 452; doing so undoes a deliberate change.  See https://github.com/nodemcu/nodemcu-firmware/pull/2214/commits/ef7a6942f8b7f3ca7c091739774d98ebf1d042dd .